tl/features/get_kg_links.py

```diff
@@ -0,0 +1,43 @@
+import pandas as pd
+import sys
+from tl.exceptions import RequiredInputParameterMissingException
+from tl.file_formats_validator import FFV
+
+def get_kg_links(kwargs):
+
+    file_path = kwargs['input_file']
+    num_kg_links = kwargs['num_kg_links']
+    label_column = kwargs['label_column']
+    score_column = kwargs['score_column']
+
+    if file_path is None:
+        raise RequiredInputParameterMissingException(
+            'One of the input parameters is required: {} or {}'.format('file_path', 'df'))
+
+    if score_column is None:
+        raise RequiredInputParameterMissingException(
+            'One of the input parameters is required: {}'.format('score_column'))
+    
+    df = pd.read_csv(file_path)
+    ffv = FFV()
+    if not(ffv.is_candidates_file(df)):
+        raise UnsupportTypeError("The input file is not a candidate file!")
+
+    topk_df = df[df['method'] == 'fuzzy-augmented'].groupby(['column','row']) \
```

Comment:
why are the results filtered on `fuzzy-augmented` only? We should filter out any results, just rank by final score irrespective of the method